controller/src/main/java/io/pravega/controller/server/rest/generated/api/HealthApiService.java

```diff
@@ -0,0 +1,32 @@
+package io.pravega.controller.server.rest.generated.api;
```

Comment:
please make sure this doesnt show up here.

Reply:
Done